editor/gui/editor_spin_slider.cpp

```diff
@@ -705,4 +706,5 @@ EditorSpinSlider::EditorSpinSlider() {
 	grabber->connect("mouse_entered", callable_mp(this, &EditorSpinSlider::_grabber_mouse_entered));
 	grabber->connect("mouse_exited", callable_mp(this, &EditorSpinSlider::_grabber_mouse_exited));
 	grabber->connect("gui_input", callable_mp(this, &EditorSpinSlider::_grabber_gui_input));
+	grabbing_spinner_speed = EditorSettings::get_singleton()->get("interface/inspector/float_drag_speed");
```

Comment:
Welp, it's crashing actually here. Try to set it in `NOTIFICATION_ENTER_TREE`..

Reply:
That's weird, it's not crashing for me. I mean I'll change it, but any idea why it crashes? Trying to understand this